integration_tests/projects/002_jaffle_shop/models/orders_forecast.sql

```diff
@@ -0,0 +1,12 @@
+{{ config(materialized='ephemeral') }}
+/*
+This table is generated by fal.
+Please run this project with `fal flow` CLI tool instead of `dbt` to
+get the correct results.
+
+Specify script dependencies, this makes fal flow run in the correct order:
+{{ ref('orders_daily') }}
+*/
+
+-- Check Python file for real behaviour
+SELECT * FROM {{ target.schema }}.{{ model.name }}
```

Comment:
is this file going to be generated by fal cli? 

can we get away by not creating this at all? what if we create this in a hidden location and point dbt to look for models in that location, we can maybe even do this without changing any configuration but by passing extra arguments to the dbt cli

Reply:
I don't think we can get away with not creating this at all, and I also believe it is important that we create it to give the users a way to **document** the table generated by fal and see it in the generated DAG.

Putting it in a different location sounds good, but I think we would need to modify dbt project configs [model-paths](https://docs.getdbt.com/reference/project-configs/model-paths) to include that.

Reply:
Another thing is that creating it in a different directory would mean creating a new schema.yml. We have to see how this looks in documentation.

Reply:
And the `fal-scripts-path` could be a "problem" for this too. Unless we just add the `sql` files to the `fal-scripts` directory.